src/targets/services/categorize.js

```diff
@@ -38,14 +38,21 @@ export const categorize = async () => {
     let filteredOperations = filters.minOperationDate
       ? operations.filter(
           e =>
-            new Date(e.date).valueOf() <=
+            new Date(e.date).valueOf() <
             new Date(filters.minOperationDate).valueOf()
         )
       : operations
 
     model = await Model.fromDocs(filteredOperations)
   }
 
+  console.log(
```

Comment:
It's easier for log readability to provide a bit of context, e.g. `console.log('Sum of occurences: ....')`